code/__DEFINES/preferences_defines.dm

```diff
@@ -63,8 +63,9 @@
 #define PREFTOGGLE_2_DANCE_DISCO			(1<<16) // 65536
 #define PREFTOGGLE_2_MOD_ACTIVATION_METHOD	(1<<17) // 131072
 #define PREFTOGGLE_2_PARALLAX_IN_DARKNESS	(1<<18) // 262144
+#define PREFTOGGLE_2_INPUT_LIST_CHOICE		(1<<19) // 524288
```

Comment:
The name isn't very descriptive, which input list method choice does this correspond to?

Reply:
TGUI by default and legacy of switched
I don't know what to call it more clearly, sorry

Reply:
PREFTOGGLE_2_INPUT_LIST_NO_TGUI is what I'd call it

Reply:
PREFTOGGLE_2_TGUI_INPUT_LIST 
I think it'll be better

Reply:
Setting it to true means there is no TGUI, right? That means the name needs to be reversed

Reply:
PREFTOGGLE_2_DISABLE_TGUI_LISTS maybe?

Reply:
Eh, sure, thats good enough